src/Sentry/Platforms/Native/buildTransitive/Sentry.Native.targets

```diff
@@ -47,9 +47,8 @@
 
   <ItemGroup Condition="'$(FrameworkSupportsNative)' == 'true' and '$(RuntimeIdentifier)' == 'linux-musl-x64'">
     <DirectPInvoke Include="sentry-native" />
-    <!-- When musl is detected, static sentry-native links to static libunwind -->
-    <LinkerArg Include="-Wl,-Bstatic -Wl,--whole-archive -lunwind -Wl,--no-whole-archive -Wl,-Bdynamic" />
-    <NativeSystemLibrary Include="lzma" />
+    <!-- When musl is detected, static sentry-native links to static libunwind, which depends on liblzma -->
+    <LinkerArg Include="-Wl,-Bstatic -Wl,--whole-archive -lunwind -Wl,--no-whole-archive -llzma -Wl,-Bdynamic" />
```

Comment:
I take it this is the guts of the fix (`-llzma` linker arg)... everything else is just so we can test this on CI?

Reply:
Yeah, exactly. This to enforce static linking, and the change to `Dockerfile` to install [xz-static](https://pkgs.alpinelinux.org/contents?file=&path=&name=xz-static&branch=v3.21&repo=main&arch=x86_64) to make `liblzma.a` available in the build environment.